packages/view-types/scatterplot/src/shared-spatial-scatterplot/ToolMenu.js

```diff
@@ -119,18 +121,18 @@ export default function ToolMenu(props) {
     <div className={classes.tool}>
       {visibleTools.pan && (
       <IconTool
-        alt="pointer tool"
+        alt={pointerIconAltText}
         onClick={() => setActiveTool(null)}
         isActive={activeTool === null}
-      ><PointerIconSVG />
+      ><PointerIconSVG aria-label={pointerIconAltText} />
```

Comment:
Is the alt text of the `IconTool` visible in accessibility tools? If so, we could use `aria-disabled` here too.